lxd/network/openvswitch/ovn.go

```diff
@@ -2047,3 +2082,154 @@ func (o *OVN) LogicalRouterPeeringDelete(opts OVNRouterPeering) error {
 
 	return nil
 }
+
+// GetHardwareAddress gets the mac address of the ovn managed network.
+func (o *OVN) GetHardwareAddress(chassisGroup OVNChassisGroup) (string, error) {
+	nameFilter := fmt.Sprintf("name=%s-lr-lrp-ext", chassisGroup)
+	data, err := o.nbctl("--data=bare", "--format=csv", "find", "Logical_Router_Port", nameFilter)
+	if err != nil {
+		return "", err
+	}
+
+	records := readCSV(data)
+	mac := ""
+	i := 0
+	for record := range records {
+		mac = record.values["mac"]
+		i++
+	}
+
+	if i != 1 || mac == "" {
+		return "", fmt.Errorf("Error getting hardware address")
+	}
+
+	return mac, nil
+}
+
+// GetMTU gets the mtu of the ovn managed network.
+func (o *OVN) GetMTU(chassisGroup OVNChassisGroup) (int, error) {
```

Comment:
I don't think we need this function as the MTU DHCP/RA options are derived in LXD and save into the bridge.mtu setting here https://github.com/lxc/lxd/blob/master/lxd/network/driver_ovn.go#L1709

Reply:
In that case shall I error if the config key isn't set? (so it is caught if the preceding logic changes)

Reply:
For now I've just returned the result of `n.getBridgeMTU()` which will be zero if the config key is not set.

Reply:
If not set, we should return 1500 which is the default value for a MTU.